core/src/edu/cornell/gdiac/main/GameController.java

```diff
@@ -140,16 +185,35 @@ public class GameController implements Screen {
    */
   public GameController() {
     level = new LevelModel();
-    complete = false;
-    failed = false;
-    active = false;
+    screenActive = false;
+    isSoundOn = !IS_MUTED;
     countdown = -1;
     timer = 100;
-    // create CollisionController, which is extended from ContactListener
+
+    // Set init game state
+    setGameState(GameState.INIT);
+
+    // Create CollisionController, which is extended from ContactListener
     collisionController = new CollisionController(level);
-    setComplete(false);
-    setFailure(false);
-    sensorFixtures = new ObjectSet<Fixture>();
+    sensorFixtures = new ObjectSet<>();
+
+    // Create menu modals
+    Batch batch = new SpriteBatch(); // Heavy class, so share among all stage menus
+    pauseMenu = new PauseMenu(new ScreenViewport(), batch);
+    successMenu = new SuccessMenu(new ScreenViewport(), batch);
+    failureMenu = new FailureMenu(new ScreenViewport(), batch);
+
+    // Set to full screens
+    pauseMenu.getViewport().update(Gdx.graphics.getWidth(), Gdx.graphics.getHeight(), true);
+    successMenu.getViewport().update(Gdx.graphics.getWidth(), Gdx.graphics.getHeight(), true);
+    failureMenu.getViewport().update(Gdx.graphics.getWidth(), Gdx.graphics.getHeight(), true);
+  }
```

Comment:
What do you mean by `set to full screens`, also I noticed that the full screen button on the window is now grayed out, did you change that in this PR? I think James is working on refactoring the game for a full screen mode so do you think he should wait to push his changes?

Reply:
It is full screen for the stage I think. So any input that occurs through clicking will be handled by either the pause menu, success menu, or failure menu (since all menus take up the entire screen). The available text and button labels are what are directly visible, though.